### DISCUSSION TITLE
Marupextension inside StringFormat

### DISCUSSION THREAD

**Discussion by trashhacker** [AvaloniaUI/Avalonia#17172] [Q&A] [answered]
is it possible to to something like that 
StringFormat={}{res:Token week} {0}}

**Comment by maxkatz6** [comment, marked as answer]
`StringFormat={res:Token weekFormat}` is possible. If you need combined, you can use MultiBinding.

**Comment by trashhacker** [comment]
how do i bind markupextensions
